custom_components/omnik_inverter/__init__.py

```diff
@@ -39,7 +38,6 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     try:
         await coordinator.async_config_entry_first_refresh()
     except ConfigEntryNotReady:
-        await coordinator.omnikinverter.close()
```

Comment:
I wonder if we should remove this rule. We may no longer reuse the session, but if something happens it would be nice to close it 🤔 Notice that this function has also been removed in the python lib in your PR.

Reply:
That function only closed the session, so after the removing of the session in the lib it was empty so I removed it.
Could leave it in as a placeholder for where to put stuff in case a close becomes relevant again.